shell/platform/darwin/ios/framework/Source/FlutterPluginAppLifeCycleDelegate.mm

```diff
@@ -32,6 +35,26 @@ @implementation FlutterPluginAppLifeCycleDelegate {
   NSPointerArray* _delegates;
 }
 
++ (void)load {
+  // Swap out `performApplication:didReceiveRemoteNotification:fetchCompletionHandler:` for
+  // `application:didReceiveRemoteNotification:fetchCompletionHandler:`.  This has to be done
+  // to avoid a potential false positive email when uploading apps to the AppStore about
```

Comment:
I don't think we have to mention an email since the developer could be informed in multiple ways, it's an App Store "Missing Push Notification Entitlement" validation failure.

Reply:
Done